app/javascript/packages/phone-input/index.ts

```diff
@@ -1,4 +1,4 @@
-import { isValidNumberForRegion, isValidNumber } from 'libphonenumber-js';
+import { isValidNumberForRegion, isValidNumber, NationalNumber } from 'libphonenumber-js';
 import 'intl-tel-input/build/js/utils.js';
 import intlTelInput from 'intl-tel-input';
 import type { CountryCode } from 'libphonenumber-js';
```

Comment:
We have an `import type ... from 'libphonenumber-js';` below, it probably makes more sense to live there.

```suggestion
import { isValidNumberForRegion, isValidNumber } from 'libphonenumber-js';
import 'intl-tel-input/build/js/utils.js';
import intlTelInput from 'intl-tel-input';
import type { CountryCode, NationalNumber } from 'libphonenumber-js';
```